view_presentation/interface/attribute_name_interface.py

```diff
@@ -0,0 +1,74 @@
+from interface import interface
+import pandas as pd
+import embedding_distance
+
+class AttributeNameInterface(interface):
```

Comment:
code structure:

- i'd include all interface instances in a "interfaces" folder. 
- I would include everything else under view_presentation, especially whatever is the main.py which is not obvious from the structure now
- if there's need for utils/ (such as to store embedding_distance) that's fine too.

There's no need to overcomplicate code structure too much as the component is quite simple. But the suggestions above will make it much easier to navigate and hence to extend and maintain.

Reply:
@raulcf did you mean ver/interfaces folder rather than ver/view_presentation/interfaces?

Rest make sense to me.